win32ss/user/ntuser/cursoricon.c

```diff
@@ -82,9 +82,16 @@ IntInsertCursorIntoList(
 {
     PPROCESSINFO ppi = pcur->head.ppi;
     PCURICON_OBJECT *ppcurHead;
+
+    //  This is hacked around to support this while at the initial system start up.
```

Comment:
Here and several other areas you mention it's hacked around, But not WHY it needs this hack in the first place.

Reply:
OK, added a short explanation comment about that as well. Thanks.